botocore/docs/bcdoc/style.py

```diff
@@ -220,7 +220,7 @@ def end_a(self):
                 if ':' in last_write:
                     last_write = last_write.replace(':', r'\:')
                 self.doc.push_write(last_write)
-                self.doc.push_write(' <%s>`_' % self.a_href)
+                self.doc.push_write(' <%s>`__' % self.a_href)
```

Comment:
There is a couple more places in the method where only a single `_` is used. Do those need to be updated as well?